### PR TITLE
Added detaching instructions, test, and setup

### PR DESCRIPTION
**Pull request by parkmichael777**

### CODE REVIEW

**Comment by bthayer2365** [comment]
From #24:
#### 1.

I don't think detaching head is quite clear what to do. Can you add more specific instructions? We don't quite have the benefit of displaying the goal state yet, so we'll need to have more specific instructions.

**Comment by parkmichael777** [comment]
Added the end goal to the instructions.

File: gitgud/levels/rampup/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from collections import OrderedDict
+
+import pkg_resources
+
+from gitgud.levels.util import BasicChallenge
+
+all_challenges = OrderedDict()
+all_challenges['detaching'] = BasicChallenge('detaching', pkg_resources.resource_filename(__name__, '_detaching/'))
```
Comment (bthayer2365) [inline gitgud/levels/rampup/__init__.py]: Can you remove all challenges except `detaching`?

**Comment by bthayer2365** [comment]
Everything looks good, but we shouldn't add the other challenges until they're finished

**Comment by parkmichael777** [comment]
Should be removed for all four PRs

**Comment by bthayer2365** [comment]
Merged!

**Pull request merged**